Win/Mineways.cpp

```diff
@@ -333,6 +338,20 @@ BOOL InitInstance(HINSTANCE hInstance, int nCmdShow)
     return TRUE;
 }
 
+
+wchar_t* stripWorldName(wchar_t* worldPath)
+{
+    // find last "/" or "\", as possible
+    wchar_t *lastSplitBackslash = wcsrchr( worldPath, '\\' )+1;
+    wchar_t *lastSplitSlash = wcsrchr( worldPath, '/' )+1;
+    wchar_t *lastSplit = worldPath;
+    if ( lastSplitBackslash > lastSplit )
+    lastSplit = lastSplitBackslash;
+    if ( lastSplitSlash > lastSplit )
+    lastSplit = lastSplitSlash;
```

Comment:
So you're relying on the fact that NULL *is* 0? I don't like it very much. Also, does it make sense to search for both `/` and `\`? Aren't you just looking for the basename?